### PR TITLE
Awesome books

### PR DESCRIPTION
**Pull request by paulinagonzalezc**
# Awesome Books
>In this pull request, we started building a basic website that allows users to add/remove books from a list. 🌟

Following the  project requirements:

* Create a collection that keeps a list of books.
* Create a function to add a new book to the collection, with title and author.
* Create a function to remove a book from the collection. 
* Display all books saved in the collection in the top part of the page.
* Make sure that when a user clicks the "Add" button:
    * A new book is added to the collection.
    * The new book is displayed in the page.
* Make sure that when a user clicks the "Remove" button:
    * The correct book is removed from the collection.
    * The correct book dissapears from the page.
* Make sure that data is preserved in the browser's memory by using localStorage.

### CODE REVIEW

**Review by rloterh — COMMENTED** [review]
Hi Team, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that is not contributed by all 3 assigned pair group members. Kindly ensure that **all** assigned pair members contribute evenly to this project, and then kindly request another code review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: app.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    `;
+
+    bookList.appendChild(content);
+  }
+
+  static deleteBook(element) {
+    if (element.classList.contains('delete')) {
+      element.parentElement.remove();
+    }
+  }
```
Comment (Gopxfs) [inline app.js:72]: According to the project requirements:
image

Your project currently deletes the book correctly from the page after clicking on the 'Remove' button, well done 😄! Please make sure to also remove it from your book collection. This means that, after removing a book from your page, it shouldn't come back after loading it again (you should update your localStorage 😉).

Your booklist:
image
Your collection: 
image

- - - - -

Booklist after removing the elements:
image
Notice that the collection remains the same:
image
And after refreshing the page, they appear again:
image

> My tip for you: you can add ids to help you identify each book and use the [filter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter) method to remove a book, given an id 😃. Good luck and keep coding!

**Review by Gopxfs — CHANGES_REQUESTED** [review]
# ♻️ STATUS: REQUIRED CHANGES ♻️

## Hi @paulinagonzalezc, @Sparktan24 👋,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights :rocket:
✔️ Nice job following the wireframe.
✔️ Books are being added without any problem.
✔️ Good work storing the collection on localStorage 😄.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 🗒️  
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Gopxfs in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BenMukebo — APPROVED** [review]
### Hi @Team, 👋 

# STATUS: Approved  🎉 

Great work on making the changes requested by a previous reviewer :1st_place_medal:  
 Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉  🎉 🎉 



### To Highlight :1st_place_medal: 
- Your Pull Request has a proper title and description. ✔️  
- No Linter errors :heavy_check_mark: 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**